packages/node_modules/@webex/redux-module-media/src/actions.js

```diff
@@ -217,6 +217,31 @@ export function listenForCalls(sparkInstance) {
   };
 }
 
+/**
+ * Checks if the machine has at least one audio or video device
+ * @returns {Promise.<Object>} {
+ *    audio: true/false,
+ *    video: true/false
+ *}
+ */
+function getSupportedDevice() {
+  return Promise.resolve().then(() => {
+    if (!navigator.mediaDevices || navigator.mediaDevices.enumerateDevices === undefined) {
+      return {
```

Comment:
Are you returning two different things? this is an object and the other return statement is returning a promise

Reply:
@lalli-flores I noticed that as well but I'm having difficult to test it locally to make sure it works. Also, that's the same approach used in the `webex-js-sdk` repo: https://github.com/webex/webex-js-sdk/blob/ba7012680e9b3a8f07f5a662dfdced485772ab43/packages/node_modules/@webex/plugin-meetings/src/media/index.js#L372

I'm not sure if that is something that is wrong there as well.

Reply:
It is returning from within a promise on line 228, so technically it is returning a promise as well.

Reply:
I got it, [from the spec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve), if the return value is a `thenable` the returned promise will follow the `then`.

Reply:
It's all good then @adamweeks?